test/project_types/script/layers/application/push_script_test.rb

```diff
@@ -42,6 +49,7 @@
     Script::Layers::Infrastructure::PushPackageRepository.stubs(:new).returns(push_package_repository)
     Script::Layers::Infrastructure::ExtensionPointRepository.stubs(:new).returns(extension_point_repository)
     Script::Layers::Infrastructure::ScriptProjectRepository.stubs(:new).returns(script_project_repository)
+    Script::Layers::Infrastructure::MetadataRepository.any_instance.stubs(:get_metadata).returns(metadata)
```

Comment:
Echoing Jacob's comment above, I think we should use a `FakeMetadataRepository` like the other stubs above.